### ISSUE TITLE
Keywords get duplicated after import

### ISSUE THREAD

**Issue by NiekHoupelyne** [TandoorRecipes/recipes#1568]
### Tandoor Version

1.0.8

### Setup

Docker / Docker-Compose

### Reverse Proxy

No reverse proxy

### Bug description

Existing keywords get duplicated (re-created) when importing from an external source.

Steps to reproduce:

- Cookbook > Import > URL > Automatic
- Enter website URL and click button to begin import
- At the bottom of the page, remove all suggested keywords
- Select one or more existing keywords having an icon (as far as I can tell, the issue only occurs for keywords that have an icon associated to them)
- check "Import all keywords, not only the ones already existing."
- click import

The recipe now gets imported. If you now check the list of keywords, you can see that the recipe was not added to the existing keywords that were selected, but new, duplicate, keywords were created instead with the same name and icon

**Comment by smilerz** [comment]
> The recipe now gets imported. If you now check the list of keywords, you can see that the recipe was not added to the existing keywords that were selected, but new, duplicate, keywords were created instead with the same name and icon

there is a database constraint that prevents duplicate keyword names.  either there is a leading/trailing space - or there is a case difference  ('Keyword' is different than 'keyword')

**Comment by NiekHoupelyne** [comment]
Yes, figured out the workaround. It's somewhat annoying though. :)
Thanks for the very quick feedback, looking forward to that new version.

**Comment by vabene1111** [comment]
I noticed when working on the importer that the keyword label is used by the keyword parser class (str(kw))

We can add this fix to the next release as well because I probably won't finish the new importer until then
